src/test/java/core/basesyntax/ArrayListTest.java

```diff
@@ -0,0 +1,82 @@
+package core.basesyntax;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class ArrayListTest {
+
+    @Test(timeout = 1000)
+    public void addElementToArrayList() {
+        ArrayList<String> arrayList = new ArrayList<>();
+        arrayList.add("Test");
+        arrayList.add("for");
+        arrayList.add("Mate");
+        Assert.assertEquals(3, arrayList.size());
+        Assert.assertEquals("Test", arrayList.get(0));
+        Assert.assertEquals("for", arrayList.get(1));
+        Assert.assertEquals("Mate", arrayList.get(2));
+    }
+
+    @Test(timeout = 1000)
+    public void addElementToArrayListByIndex() {
```

Comment:
@ymilenin do you considering the use case when you have for example 4 elements in your array list and you are trying to add an element with index = 7?